doc/Architecture-overview/index.md

```diff
@@ -10,156 +10,157 @@ sort_idx: 15
-or a [clustered deployment]({{root_url}}Administration-guide/System-installation/Cluster-setup/).
+An [endpoint (EP)]({{root_url}}Glossary/#endpoint-ep) is an abstraction that represents a separately managed entity within a Kaa deployment.
+Practically speaking, an endpoint is a specific [Kaa client]({{root_url}}Glossary/#kaa-client) registered (or waiting to be registered) within a Kaa instance.
+Depending on the use case, different level physical entities can be considered endpoints.
+In an industrial setting, a single air quality sensor can represent an individual endpoint, while in a fleet tracking application, a truck (despite carrying
+on board multiple sensors that report data) may be a more appropriate entity to be declared as an endpoint.
 
-An *application* in Kaa defines the set of data models, the corresponding flows among the endpoints and Kaa server, and processing rules.
-Kaa applications are agnostic to the specific target platform, operating system, or the client software implementation.
-For example, two firmware implementations for a pressure sensor will differ between Arduino and STM32 platforms, yet will be considered the same application
-in Kaa, as long as they report identically structured telemetry data.
+![Logical concepts](attach/logical-concepts.png)
 
-The Kaa platform is *multi-tenant*, and may support multiple independent business entities out of a single instance.
-As illustrated in the preceding diagram, applications belong to tenants, while endpoints register within applications.
+To distinguish endpoints by different parameters and properties, rather than use one ID, Kaa uses [endpoint profiles]({{root_url}}Glossary/#endpoint-profile-client-side-server-side).
 
-An *endpoint* (also commonly shortcut as *EP*), on its part, is an abstraction that represents a separate managed entity within a Kaa deployment.
-Practically speaking, an endpoint is a specific Kaa client registered (or waiting to be registered) within a Kaa instance.
-Depending on the use case, different level physical entities may be considered as endpoints.
-In the industrial setting, a single air quality sensor may represent an individual endpoint, while in a fleet tracking application, a truck (despite carrying
-on board multiple sensors reporting data) may be a more appropriate entity to be declared as an endpoint.
+Endpoint profile is a custom structured data set that describes characteristics of a specific endpoint within an application.
+Every endpoint profile comprises the *client-side*, *server-side*, and *system* parts.
+Initial values for the client-side part are specified by the the client developer using data schemas for the [endpoint SDK]({{root_url}}Glossary/#endpoint-sdk).
+Then, the client-side endpoint profile is generated during registration of a new endpoint.
+The server-side and system parts of the profile contain data about the endpoint properties that are controlled by your server-side applications.
```

Comment:
I propose to replace this line with "The server-side and system parts of the endpoint profile data are managed by the Kaa server."